src/main/java/org/gridsuite/filter/server/dto/LineFilter.java

```diff
@@ -36,34 +37,32 @@ public EquipmentType getEquipmentType() {
     String substationName2;
 
     @Schema(description = "Countries1")
-    private SortedSet<String> countries1;
+    private Set<String> countries1;
```

Comment:
SortedSet for countries is replaced by Set, but it's not the case in other dto ?

Reply:
Missed to revert this one too.

Reply:
Reverted